<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>From</class>
 <widget class="QWidget" name="From">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="1" column="0">
    <widget class="QScrollArea" name="contentArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="Parameter_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>284</width>
        <height>665</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="contentLayout" stretch="0,10,1,10,1,10,1,10,0,0,0,1,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>9</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
font: 87 9pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Configure Parameter</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1" alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_pixel_2">
           <property name="text">
            <string>1.85um</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_RI_2">
           <property name="text">
            <string>1.52</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Refractive Index</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_mag_2">
           <property name="text">
            <string>20X</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_64">
           <property name="text">
            <string>Magnification</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_wavelenth_2">
           <property name="text">
            <string>640nm</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_65">
           <property name="text">
            <string>Wavelength</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_66">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Pixel Size</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_67">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>9</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
font: 87 9pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Camera Parameter</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_15">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="1">
          <widget class="QLabel" name="label_bit_4">
           <property name="text">
            <string>16 bit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_gain_4">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_68">
           <property name="text">
            <string>Gain</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_69">
           <property name="text">
            <string>Bit Depth</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_70">
           <property name="text">
            <string>Gamma</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_gamma_4">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_71">
           <property name="text">
            <string>Image Size</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_shape_4">
           <property name="text">
            <string>3000x4000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_72">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>9</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
font: 87 9pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Reconstruction</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_16">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_73">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_step_2">
           <property name="text">
            <string>1um</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_end_2">
           <property name="text">
            <string>0um</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_74">
           <property name="text">
            <string>Step</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_75">
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_enable_2">
           <property name="text">
            <string>True</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_76">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_start_2">
           <property name="text">
            <string>0um</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_77">
           <property name="text">
            <string>Sharpness</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_sharpness_2">
           <property name="text">
            <string>Fourier Analysis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_78">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>9</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
font: 87 9pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Process Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_17">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="7" column="1">
          <widget class="QLabel" name="label_Gaussian_size_2">
           <property name="text">
            <string>80</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Hologram start</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_holo_strat_2">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_leveling_2">
           <property name="text">
            <string>Gaussian Blur</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Total Holograms</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_79">
           <property name="text">
            <string>Leveling Method</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_GB_size_2">
           <property name="text">
            <string>Gaussian Size</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_holo_total_2">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>ROI Enable</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_roi_enable_2">
           <property name="text">
            <string>False</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>FFT filter Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>FFT Quadrant</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_quadrant_2">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_filter_rate_2">
           <property name="text">
            <string>1.2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_80">
           <property name="text">
            <string>FFT Filter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_FFT_filter_2">
           <property name="text">
            <string>Hann</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_81">
           <property name="text">
            <string>Filter Rate</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Expansion</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_expansion_2">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Confirm">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1600</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Confirm and Go to Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_82">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>9</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 9pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>What to Save</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="3" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Refocused Volume</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Height Map</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_Height_2">
           <property name="text">
            <string>True</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Phase Map</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Wrapped Phase</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_Phase_2">
           <property name="text">
            <string>False</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_Wrapped_phase_2">
           <property name="text">
            <string>False</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_refocused_volume_2">
           <property name="text">
            <string>False</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="toggleButton">
     <property name="text">
      <string>STEP1: Set Parameter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
